app/views/shared/_accordion.html.erb

```diff
@@ -4,24 +4,26 @@
 
     <div class="card">
     <div class="card-header d-flex justify-content-between" id="heading<%=index.humanize.capitalize%>">
+    <div class="flex-grow-1">
       <h2 class="mb-0">
         <button class="btn btn-link btn-block text-left" type="button" data-toggle="collapse" data-target="#collapse<%=index.humanize.capitalize%>" aria-expanded="true" aria-controls="collapse<%=index.humanize.capitalize%>">
-          <%= favorite_topic.topic.title.capitalize %>
+        <h3><%= favorite_topic.topic.title.capitalize %> </h3>
```

Comment:
guys, we have a h3 inside a button inside a h2

```
h3
    button
        h2

```

You can use some HTML validators to help check if things make sense or not like this one https://validator.w3.org/#validate_by_input

@Sajakhtar @khamzahabibullah

Reply:
Issue addressed